paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandlerTest.kt

```diff
@@ -67,17 +69,20 @@ class FlowControllerConfigurationHandlerTest {
 
     @Test
     fun `configure() should run initial configuration`() = runTest {
-        val countDownLatch = CountDownLatch(1)
+        val completable = CompletableDeferred<Throwable?>()
```

Comment:
This refactor makes a lot of sense to me. I should have done this to begin with!

I would feel more comfortable if we broke this refactor out to a separate PR that we land before the rest of this though.

Reply:
Actually going with Turbines, right [here](https://github.com/stripe/stripe-android/pull/6494).